lib/rubygems_plugin.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```

Comment:
Can we add comments to explain when and by what this file will be used?

Reply:
I added comment.
Can you check that?